MIPS/mip-29.md

```diff
@@ -1,61 +1,85 @@
 ---
 mip: 29
-title: Decide on the Future for mStable
-status: WIP
+title: Adjustment of Builder SubDAO Mandate
+status: Proposed
 author: Cam Soulsby (@camsoulsby), James Simpson, 0xloth (@LDeroccha), mZeroNine (@mzero2nine) 
-discussions-to:
-created: 2023-01-31
+discussions-to: https://forum.mstable.org/t/mip-29-adjustment-of-builder-subdao-mandate/982
+created: 2023-02-06
```

Comment:
created maybe should have the initial creation date?

Reply:
That's right, good catch!